portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -87,11 +87,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     
       Gson gson = new Gson();
       response.setContentType("application/json;");
-
-      List<BlogMessage> limitedBlogMessages = new ArrayList<>();
-      for (int i = 0; i < numberOfCommentsToDisplay; i++) {
-        limitedBlogMessages.add(allBlogMessages.get(i));
-      }
+      
       response.getWriter().println(gson.toJson(limitedBlogMessages));
```

Comment:
Hmm I am confused about what "limitedBlogMessages" is. why do we need to remove 90-94?

Reply:
Umm I never used "limitedBlogMessages". I think it happened when I tried to merge master into my branch to resolve conflicts after i made the PR. When I took 90-94 out i forgot to also change that part to "loadedBlogMessages".

Reply:
But we don't need those lines because the BlogHashMap sends the requested amount of messages.
Those lines were written before the BlogHashMap implementation and were already in master. So when i merged master into my branch, things got mixed up

Reply:
So when you merge and resolve, you would normally either try make a judgement call yourself and remove it, or ask the original author about what is going on, confirm with them if you should just remove it. In the end of the day, this is a shared codebase (everyone has to maintain it).